brizo/routes.py

```diff
@@ -688,10 +688,22 @@ def compute_start_job():
         if not asset_urls:
             return jsonify(error=f'cannot get url(s) in input did {did}.'), 400
 
+        categories = []
+
+        # ! will gather categories and unify them across ALL DIDS
+        try:
+          for category in asset.as_dictionary()["service"]:
+            attrs = category["attributes"]
+            extra = attrs["additionalInformation"]
+            categories.extend(extra["categories"])
+        except Exception:
```

Comment:
Codacy found an issue: [Try, Except, Pass detected.](https://app.codacy.com/manual/ocean-protocol/brizo/pullRequest?prid=5749023)

Reply:
ahh, haven't re-specified the condition after removing pymongo. will change in a sec